bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/AuthFilter.java

```diff
@@ -63,6 +71,25 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                         Authentication auth = jwtHelper.verifyAndParseJwtAccessToken(authParts[1]);
                         requestContext.setSecurityContext(new JwtSecurityContext(auth));
                         return;
+                    } else if ("Basic".equals(authParts[0])) {
```

Comment:
A small suggestion here - it might be worth to use `equalsIgnoreCase` as some tools can send auth type lowercase.